.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+  # Enable version updates for npm
+  - package-ecosystem: "npm"
+    # Look for `package.json` and `lock` files in the `root` directory
+    directory: "/"
+    # Check the npm registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+    allow:
+    - dependency-type: production
+    commit-message:
+        prefix: "[DEPS]"
+    # Specify labels for npm pull requests
+    labels:
+        - "npm"
+        - "dependencies"
+    target-brach: "master"
+    ignore:
+          # For express, ignore all updates. Since can be breaking
+        - dependency-name: "express"
+          # For Loadash, ignore all updates. Since can be breaking
+        - dependency-name: "loadash"
```

Comment:
Thanks for this, can I ask why we are ignoring these? Also, would you mind adding a newline to the end of the file. Thanks!

Reply:
@jenkoian Added End of Line. We can Ignore few dependency since sometimes bumping dependency version leads to some unknown functionality breakdown.

Reply:
I think there's a typo in `lodash` but anyways I think we should include these and if we decide the number of breaking changes becomes annoying/prohibitive we can ignore in the future.

Reply:
Sure. Updated the PR